### DISCUSSION TITLE
JDBC Metrics instrumentation from opentelemetry-jdbc library

### DISCUSSION THREAD

**Discussion by robsonfirmino** [open-telemetry/opentelemetry-java-instrumentation#11854] [Q&A]
I am implementing manual JDBC instrumentation using [opentelemetry-jdbc library](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jdbc/library) in my project. I've noted that there is no metrics implementation in the current library ([2.6.0-alpha-SNAPSHOT](https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-jdbc/2.6.0-alpha)). 
Is there some initiative to implement [database metrics](https://opentelemetry.io/docs/specs/semconv/database/database-metrics/) in the opentelemetry-jdbc library?

**Comment by trask** [comment]
Yes! @heyams is currently planning to implement the latest database semconv in the jdbc instrumentation (behind a flag for now until database semconv is marked as stable)

**Reply by robsonfirmino** [reply]
@trask Thanks  for answer! 
@heyams  Do you have some branch started or are you waiting 'semconv' stable to start coding jdbc metrics?
I'm asking this because maybe I can contribute from some way.

**Reply by RafaelCavazza** [reply]
Same here. Is there a way to enable these metrics as a preview?
They would be helpful to our observability stack.

**Comment by matheus-meneses** [comment]
Same here
